k8s-ci/k8s_crds/test

```diff
@@ -16,6 +16,11 @@ exit_err() {
    cat ${log};
    rm -rf ${log};
 
+
+  # Snapshot of workloads
+  kubectl --namespace "$(current_namespace)" get pods || true
+  kubectl --namespace "$(current_namespace)" get events --field-selector type=Warning || true
```

Comment:
Add debug echos to this - it's hard to tell what you might be looking at just from output of a kubectl command